chainer/functions/array/flatten.py

```diff
@@ -13,12 +13,34 @@ def backward(self, inputs, grads):
 
 
 def flatten(x):
-    """Flatten a given array.
+    """Flatten a given array into one dimension.
```

Comment:
Do you think we should mention the situation that input array is a scalar (i.e. shape `()`)? In that case, the resulting array has a shape `(1,)`. I think it is better to add it to the document. But I am also thinking users could think it is natural and need not to treat as an exceptional case.